cli/cli.go

```diff
@@ -0,0 +1,69 @@
+// Package cli implements some simple shared CLI flag types.
+package cli
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	pb "github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2"
+	"github.com/peterebden/go-cli-init"
+)
+
+var log = cli.MustGetLogger()
+
+var actionRe = regexp.MustCompile("([0-9a-fA-F]+)/([0-9]+)/?")
```

Comment:
nit: Don't really need a regex here. We could just strings.split on /

Reply:
Yeah but the regex also handles the restricted character sets; I don't think there's a trivial way of doing that.

I'd do something else if performance was an issue but it's only CLI parsing so it's not like we're gonna need to consume thousands of these.